### ISSUE TITLE
The Tabby turn into black after flash window

### ISSUE THREAD

**Issue by maswangy** [Eugeny/tabby#7776]
Tabby Version: 1.0.188
![image](https://user-images.githubusercontent.com/8416758/211708030-68a4e67a-e50a-49f8-ae2f-a89810afe280.png)

I hadn't use the Tabby for a while. Two days ago I opened Tabby and it turn into black window completely after flash window.You cannot move it but you can resize it. Nothing else response. The only way to close it is kill it in task manager.
![image](https://user-images.githubusercontent.com/8416758/211708527-f0cd8a2e-38da-4325-8a30-be23f4193c5d.png)

I forgot the initial version of tabby that I installed and I unistalled it and tried 1.0.188  x64  portal and setup version. the results are the same.

I am not sure this is caused by windows updates ( I remembered that in Dec window auto updated once).

**Comment by zhaowq32** [comment]
I also encountered this problem.
Tabby version 1.0.188 
macos version 12.6.2

**Comment by max1c** [comment]
Yes, it keeps flashing black for me like this as well and freezes randomly sometimes on Mac. Has become unusable after the latest update.

**Comment by MCazaly** [comment]
Also experiencing this on Ubuntu 22.04.1 LTS. It occurs with versions 1.0.185-1.0.1.188. The latest version that works as expected for me is 1.0.184.

**Comment by xinferum** [comment]
Same problem.
Tabby version: 1.0.188
Windows version: Windows 10 Pro 22H2, build 19045.2486

**Comment by Eatkarus** [comment]
same problem，and it's frequent
platform system：Windows 11
tabby：V1.0.215

![bug](https://github.com/user-attachments/assets/b4e450f4-14b0-4520-8f73-ef536315e7b3)
